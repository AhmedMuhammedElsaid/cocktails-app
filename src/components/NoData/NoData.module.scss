@import '../../scss//variables.scss';

.NoData {
  padding: 70px 0;
  background-color: $mainWhite;
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
  text-transform: uppercase;
}
