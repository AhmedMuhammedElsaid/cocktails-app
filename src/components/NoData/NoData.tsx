import React, { FunctionComponent } from "react";
import styles from "./NoData.module.scss";

const NoData: FunctionComponent = () => {
  return (
    <div className={styles.NoData} data-testid="no-data-testid">
      No Data Found
    </div>
  );
};

export default NoData;
