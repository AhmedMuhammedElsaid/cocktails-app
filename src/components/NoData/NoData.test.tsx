import { render } from "@testing-library/react";
import NoData from "./NoData";

describe("test no data component", () => {
  it("renders without error", () => {
    const { getByTestId,asFragment } = render(<NoData />);
    expect(getByTestId("no-data-testid")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot()
  });
});
