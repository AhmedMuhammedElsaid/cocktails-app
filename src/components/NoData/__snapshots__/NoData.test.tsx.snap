// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test no data component renders without error 1`] = `
<DocumentFragment>
  <div
    class="NoData"
    data-testid="no-data-testid"
  >
    No Data Found
  </div>
</DocumentFragment>
`;
